rules/namingConventionRule.ts

```diff
@@ -65,6 +66,7 @@ enum TypeSelector {
     genericTypeParameter = Types.type | Types.genericTypeParameter,
     enum = Types.type | Types.enum,
     enumMember = property | Types.enumMember,
+    accessor = Types.member | Types.method | Types.accessor,
```

Comment:
I agree about that. 

I think `accessor` is method because we define that like method.
But maybe someone would think `accessor` is property because we use `accessor` like property.

I will make commit which put `accessor` extending member, not method.

Reply:
@ajafff Updated.